qa/mixed-cluster/build.gradle

```diff
@@ -64,6 +64,7 @@ for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible) {
       numberOfNodes = 4
 
       setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
+      setting 'opensearch.experimental.feature.search_pipeline.enabled', 'true'
```

Comment:
Does this imply that it is enabled by default?

Reply:
It's enabled for the BWC tests, because otherwise the BWC tests won't pass.